localboot/main.go

```diff
@@ -19,6 +19,7 @@ var (
 	flagBaseMountPoint = flag.String("m", "/mnt", "Base mount point where to mount partitions")
 	flagDryRun         = flag.Bool("dryrun", false, "Do not actually kexec into the boot config")
 	flagDebug          = flag.Bool("d", false, "Print debug output")
+	flagConfigNo       = flag.Int("config", -1, "Specify the bootconfiguration to boot. It is the order of the menuentries in grub.cfg")
```

Comment:
I think a better default would be `0`. In this way the current behaviour is unchanged, and you don't have to check if the config number is < 0.

nit: maybe a more descriptive name could be `flagConfigIdx`, and the description could become "Specify the index of the configuration to boot. The order is determined by the menu entries in the Grub config"

Reply:
naming sound good, changed it